$OpenBSD: patch-Makefile_PL,v 1.3 2019/01/25 22:32:04 sthen Exp $

#1, fix perl 5.26.0+

#2, https://rt.cpan.org/Public/Bug/Display.html?id=117793

Also, add a zero to defined constants. So if the define is empty, the code
still makes sense.

Index: Makefile.PL
--- Makefile.PL.orig
+++ Makefile.PL
@@ -1,6 +1,7 @@
 # Makefile.PL for Perl module WWW::Curl
 # Check out the README file for more information.
 
+BEGIN { push @INC, '.'; }
 use inc::Module::Install;
 
 name			'WWW-Curl';
@@ -127,7 +128,7 @@ if (!defined($curl_h)) {
     close H;
 
     for my $e (sort @syms) {
-       if($e =~ /(OBSOLETE|^CURL_EXTERN|_LAST\z|_LASTENTRY\z)/) {
+       if($e =~ /(OBSOLETE|^CURL_EXTERN|^CURL_STRICTER\z|_LAST\z|_LASTENTRY\z)/) {
           next;
        }
        my ($group) = $e =~ m/^([^_]+_)/;
@@ -190,7 +191,7 @@ HERE2
             if ($next_initial eq $initial) {
 
                 print CURL_XS
-"            if (strEQ(name, \"$remainder\")) return "."$groupref->{$option};\n";
+"            if (strEQ(name, \"$remainder\")) return "."$groupref->{$option}+0;\n";
 
                 $count++;
             }
